solitaire.rb

```diff
@@ -1,4 +1,35 @@
 class Solitaire
```

Comment:
If you are starting over, why isn't this a new file? Or did I misunderstand the commit message?